worker.go

```diff
@@ -456,7 +457,7 @@ func (w *Worker) ImportMetricGRPC(other *metricpb.Metric) (err error) {
 	w.mutex.Lock()
 	defer w.mutex.Unlock()
 
-	key := samplers.NewMetricKeyFromMetric(other)
+	key := samplers.NewMetricKeyFromMetric(other, []matcher.TagMatcher{})
```

Comment:
Is this supposed to be hardcoded to an empty slice?

Reply:
Yes, this is in veneur, not veneur-proxy.